### DISCUSSION TITLE
router4的文档关于函数式导航的修改

### DISCUSSION THREAD

**Discussion by nanfb** [vuejs/router#1734] [Help and Questions] [answered]
我观察到在 https://github.com/vuejs/router/blob/main/packages/router/CHANGELOG.md#414-2022-08-22 这里修改了不能使用params传参，但是文档没有修改，也没有给出替代方案，这是不及时的，会造成困扰

**Comment by posva** [comment, marked as answer]
It's actually been mentioned for a very long time in Vue router 3 docs and Vue Router 4: https://router.vuejs.org/guide/essentials/dynamic-matching.html
The params are always connected to the URL and nowhere in docs you will find an example of passing artificial params
